google/cloud/spanner_dbapi/connection.py

```diff
@@ -124,6 +125,24 @@ def spanner_client(self):
         """
         return self._instance._client
 
+    @property
+    def schema_name(self):
```

Comment:
This property should be named `current_schema`.

Reply:
Done